python/paddle/v2/framework/layers.py

```diff
@@ -259,17 +287,15 @@ def sequence_conv(input,
         type='sequence_conv',
         inputs={
             'X': [input],
-            'Filter': filter,
+            'Filter': [filter],
         },
         outputs={"Out": pre_bias},
         attrs={
             'context_stride': stride,
             'context_start': 0,
             'context_length': filter_size
         })
-
-    pre_act = helper.append_bias_op(pre_bias)
-    return helper.append_activation(pre_act)
```

Comment:
Why remove activation？

Reply:
I think we shall retain it. If users don't want to use it, they can set `act` as `None`.